libc/src/string/strcmp.c

```diff
@@ -0,0 +1,10 @@
+#include <string.h>
+
+int strcmp(const char* str1, const char* str2) {
```

Comment:
This function is still not correct, it returns 0 if str1 terminates early but str1 and str2 doesn't match.

Reply:
Now it should always give correct return value